test/predictions/predictions_test.exs

```diff
@@ -372,7 +368,7 @@ defmodule Predictions.PredictionsTest do
               }, _} = get_all(feed_message, @current_time)
     end
 
-    test "include predictions with low uncertainty" do
+    test "include non reverse predictions" do
       reassign_env(:filter_uncertain_predictions?, true)
```

Comment:
drive-by: Could we rename this flag, since we're otherwise removing the concept of "uncertainty" from the code? Maybe `filter_reverse_predictions?`. (Curious why this is configurable in the first place.)

Reply:
Yeah, we should definitely rename it but I was thinking maybe in a small follow-up task or something since we'd want to change that in the devops repo as well. In terms of why it's configurable, I'd guess it's because back then RTS was running on opstech3 and we would often take advantage of flags like this to quickly change up flags to adjust what RTS was doing. It was added in a really old PR a few years back and may not be necessary anymore. https://github.com/mbta/realtime_signs/pull/369

Reply:
Hm, yeah, maybe more useful initially as a "feature flag" in the time immediately after the feature was added. I think we could outright remove it now, if @panentheos agrees.

Reply:
Yeah you're probably right. The feature flag isn't something we'd likely prefer to use in a situation where we want to filter out any sort of prediction type at large anyway since that requires infra to get involved these days. In the more recent past when we've filtered out terminal/reverse predictions on some heavy rail lines, we just introduced a temporary code change and then revert it when predictions were trustworthy again.